sklearn/tree/_tree.pyx

```diff
@@ -1149,14 +1136,14 @@ cdef class Tree:
         shape[0] = <cnp.npy_intp> self.node_count
         cdef cnp.npy_intp strides[1]
         strides[0] = sizeof(Node)
-        cdef cnp.ndarray arr
+        cdef Node[::1] arr
         Py_INCREF(NODE_DTYPE)
-        arr = PyArray_NewFromDescr(<PyTypeObject *> cnp.ndarray,
+        arr = PyArray_NewFromDescr(<PyTypeObject *> np.ndarray,
                                    <cnp.dtype> NODE_DTYPE, 1, shape,
                                    strides, <void*> self.nodes,
                                    cnp.NPY_DEFAULT, None)
```

Comment:
```suggestion
                                   cnp.NPY_ARRAY_DEFAULT, None)
```

Ref: https://github.com/numpy/numpy/blob/8ace612c3b1bbf0bd23f10556492908afdc97207/tools/swig/numpy.i#L52

Reply:
Thank you for the suggestion!